### ISSUE TITLE
Members set and not defined in externs are renamed and do not produce warnings 

### ISSUE THREAD

**Issue by phistuck** [google/closure-compiler#622]
```
var /** @type {HTMLFormElement} */ eForm =
       /** @type {HTMLFormElement} */ (document.querySelector("form"));
eForm.acceptCharset = "utf-8"; // Defined in externs.
eForm.undefinedProperty = true; // Not defined in externs.
```

Compiles to -

```
var a=document.querySelector("form");a.acceptCharset="utf-8";a.a=!0;
```

Note the `a.a` instead of `a.undefinedProperty`.
While this is may be desirable in some cases, in other cases, it might just be a typo. For example, `eForm.novalidate` (incorrect, but the name is similar to the HTML content attribute) versus `eForm.noValidate` (correct, this is the IDL name).
Same goes for `CSSProperties` where CSS properties are not always defined in the built in externs and can lead to subtle (or, in the best case, harsh) runtime issues.

I guess the fact that Closure Compiler renames the properties is fine, but a warning for those properties (setting properties of interfaces defined in externs, where the property being set is not defined in externs) should be added (perhaps an optional one).

I have been burned by that a lot of times already (I obviously submit pull requests for the missing built in properties, but if these are your custom externs, for third party code or something, then you simply would not know until you get a bug about it). :(

**Comment by ChadKillingsworth** [comment]
I believe this is now addressed by the `strictMissingProperties` warnings guard.
